components/org.wso2.bfsi.consent.management.common/src/main/java/org/wso2/bfsi/consent/management/common/config/ConsentManagementConfigParser.java

```diff
@@ -51,6 +51,8 @@ public class ConsentManagementConfigParser {
     private static final Object lock = new Object();
     private static volatile ConsentManagementConfigParser parser;
     private static final Map<String, Object> configuration = new HashMap<>();
+    private OMElement rootElement;
```

Comment:
Why we had to move this to the class level?

Reply:
This will require to get the configs such as consent management auth steps